tests/e2e/SMS/vonage.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Tests\E2E;
+
+use Utopia\Messaging\Adapters\SMS\Telnyx;
+use Utopia\Messaging\Messages\SMS;
+
+class TelnyxTest extends Base
+{
+    /**
+     * @throws \Exception
+     */
+    public function testSendSMS()
+    {
+        // $sender = new Telnyx(getenv('TELNYX_API_KEY'));
+
+        // $message = new SMS(
+        //     to: ['+18034041123'],
+        //     content: 'Test Content',
+        //     from: '+15005550006'
+        // );
+
+        // $result = \json_decode($sender->send($message), true);
+
+        // $this->assertEquals('success', $result["type"]);
+
+        $this->markTestSkipped('Requires a business account, and to contact them first before getting access.');
```

Comment:
You can sign up for an account here: https://ui.idp.vonage.com/ui/auth/registration